### ISSUE TITLE
Adding destination charging waypoint doesn't provide charging type selection, EVTO unable to find superchargers after segmenting

### ISSUE THREAD

**Issue by EVGrokker** [aesculus/EVTO-App-Feedback#271]
Trip: 1. Winnemucca NV, [8vdc11g1s9ld4ptjmynibm]

This is the trip I'm using for the tutorial under development in issue #267. In step 6, when searching for the intended destination charging/overnight stop at the Legacy Inn in Price UT, there is no option to allow me to indicate that this waypoint has charging capability.

So, I add it anyway. Following the steps as written so far, I can segment at the Legacy Inn and then set the charger type to NEMA 14-50. Awkward, but at least it works.

For some reason, at this point, EVTO is unable to optimize segment 3, finding no superchargers at all. Oops.

Touching the lightning bolt button shows plenty of superchargers on the map.

**Comment by aesculus** [comment]
I'll check it out. I have another trip from Helsinki that is behaving the same way. Plenty of SOC to get to the next charger and if entering it manually, the rest of the trip optimizes fine. Maybe the same bug here.

**Comment by aesculus** [comment]
I just tried this in V1.1.4 (7).  One thing I noticed was that the first segment optimization came up with only one charger and that was not enough. Doing it a second time it was OK. I think this may be related to the fact that I use a conservative (a little more than the car's EPA rated efficiency) on the first attempt and then when you do it again, I use the efficiency it calculated with the weather.

The second leg ended up not changing any of the chargers.

The third let came out OK.

![image](https://cloud.githubusercontent.com/assets/283364/25214718/a874ce22-254d-11e7-9e0c-996814ada948.png)
